lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -911,79 +889,6 @@ sub send_to_beta :Chained('base') :PathPart('send_to_beta') :Args(0) {
     return $c->forward($c->view('JSON'));
 }
 
-sub asana :Chained('base') :PathPart('asana') :Args(0) {
```

Comment:
Do we want to leave these subs in place? Or will they be moved out to a script or something at some point?

Wondering if they'll be used again in this controller.